kyverno/kyverno
# add: http/https regex to kyverno CLI

## Related issue

Fixes #2031 

- Also addresses minor typos


## What type of PR is this

/kind cleanup

## Proposed Changes

- Added regex

To test:
Create two files:

`test.yaml`:

```yaml
name: test http-ingress
policies:
    - https-ingress.yaml
resources:
results:
```

`https-ingress.yaml`:

```yaml
apiVersion: kyverno.io/v1
kind: ClusterPolicy
metadata:
  name: require-labels
spec:
  validationFailureAction: enforce
  rules:
  - name: check-for-labels
    match:
      resources:
        kinds:
        - Pod
    validate:
      message: "label 'app.kubernetes.io/name' is required"
      pattern:
        metadata:
          labels:
            app.kubernetes.io/name: "?*"

```
Results:
![image](https://user-images.githubusercontent.com/31009634/122933709-81b5f580-d38c-11eb-911e-603d7b23b4ce.png)

## Checklist



- [] I have added tests that prove my fix is effective or that my feature works.
  - [] I have added or changed [the documentation](https://github.com/kyverno/website) myself in an existing PR and the link is:
  
  
  - [x] I have raised an issue in [kyverno/website](https://github.com/kyverno/website) to track the doc update and the link is:
